html/about.html

```diff
@@ -0,0 +1,105 @@
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+    <section id="about" class="about">
+        <div class="main-about">
+            <span class="greeting">"Hello! Sharing world"</span>
+            <h2 class="title">International JavaScript Conference 2022</h2>
+            <span class="line">____</span>
+            <p>The JavaScript world is constantly evolving and there’s 
+                something new to learn every day. We at International 
+                JavaScript Conference want you 
+                to be a part of that change and stay on top of the latest trends.
+                From Angular and React to WebAssembly; from Progressive Web Apps to JAMstack – international experts share insights on the present and future JavaScript. 
+                Join us to broaden your knowledge and network with other JS enthusiasts.
+            </p>
+            <h3>Please contact us per Email for any further questions 
+                about International JavaScript Conference 2022!</h3>
+            <h4>ssuessenberger@javascript-conference.com</h4>
+        </div>
+    </section>
+    <section>
+        <ul  class="about-logo">
+            <li>International JavaScript Conference</li>
+            <li><span>___</span></li>
+            <li>The JavaScript world is constantly evolving and there’s something new to learn every day. 
+                We at International JavaScript Conference want you to be a part of that change 
+                and stay on top of the latest trends.
+                From Angular and React to WebAssembly; from Progressive Web Apps to JAMstack – international experts share insights on the present and future JavaScript.
+                 Join us to broaden your knowledge and network with other JS enthusiasts.
+            </li>
+            <li><div class="logo-img"></div></li>
+        </ul>
+    </section>
+    <section>
+        
+        <ul  class="past-conf">
+            <li>See the International JS Conference</li>
+            <li><span>___</span></li>
+            <li>Take a look at the two International JavaScript Conference witch took place in London and Munich</li>
+            <li><a href="#" class="img-london"><div class="bg-transparent">
+                <h2>2019</h2>
+                <h3>International JS Confference 2019 in London</h3>
+            </div></a>
+            <a href="#" class="img-munich"><div class="bg-transparent">
+                    <h2>2018</h2>
+                <h3>International JS Confference 2018 in Munich</h3>
+                </div></a>
+            </li>
+        </ul>
+    </section>
+</main>
+<footer >
+    <ul class="footer-about">
+        <li>International JavaScript Conference</li>
+        <li>© Copyright 2021 S&S Media, All Rights Reserved</li>
+    </ul>
+</footer>
```

Comment:
- [ ] Following the project requirements, the logo should be added to the footer. Kindly add the same logo you used in the header to the footer.
<img width="947" alt="image" src="https://user-images.githubusercontent.com/77190203/157994295-220d9f52-e0a3-4587-8856-9e5c1d84179e.png">